ansys/dpf/core/plotter.py

```diff
@@ -83,11 +110,47 @@ def show_figure(self, **kwargs):
         show_axes = kwargs.pop("show_axes", None)
         if show_axes:
             self._plotter.add_axes()
-        return self._plotter.show()
+        kwargs_in = self._sort_supported_kwargs(
+            bound_method=self._plotter.show,
+            **kwargs
+            )
+        return self._plotter.show(**kwargs_in)
 
 
 class DpfPlotter:
+    """DpfPlotter class. Must be used in order to plot
+    results over a mesh.
+
+    The DpfPlotter is a PyVista dependant object.
```

Comment:
@anslpa Since the idea behind the DpfPlotter is to create an API for plotting DPF entities (which is currently using pyVista, but which could use something else), I think that `The DpfPlotter is a PyVista dependant object` is a bit too strong